doc/api.md

```diff
@@ -1,122 +1,143 @@
 # IoT Agent API
 
-<!-- TOC WIP -->
+<!-- TOC -->
+
+-   [Preface](#preface)
+-   [Topics](#topics)
+    -   [Terminology](#terminology)
+    -   [IoT Agent information model](#iot-agent-information-model)
+        -   [Config groups](#config-groups)
+        -   [Devices](#devices)
+    -   [Entity attributes](#entity-attributes)
+    -   [Advice on Attribute definitions](#advice-on-attribute-definitions)
+        -   [Reuse of attributes names](#reuse-of-attributes-names)
+        -   [Reuse of attribute types](#reuse-of-attribute-types)
+        -   [How to specify attribute Units of Measurement](#how-to-specify-attribute-units-of-measurement)
+    -   [Multitenancy, FIWARE Service and FIWARE ServicePath](#multitenancy-fiware-service-and-fiware-servicepath)
+-   [API Routes](#api-routes)
+    -   [Config group API](#config-group-api)
+        -   [Config group datamodel](#config-group-datamodel)
+        -   [Config group operations](#config-group-operations)
+            -   [Retrieve config groups `GET /iot/services`](#retrieve-config-group-get-iotservices)
```

Comment:
Links seems to be broken.

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1336/commits/7b5379132d70466470812d65f22f04fc594e526b